bundle/Resources/config/services.yaml

```diff
@@ -55,6 +55,16 @@ services:
         tags:
             - { name: console.command, command: 'ngsite:profiler:clear-cache' }
 
+    ngsite.command.test_multiprocess:
+        class: Netgen\Bundle\SiteBundle\Command\TestMultiprocessCommand
+        autowire: false
```

Comment:
I believe these two are `false` by default.

Reply:
C/p mistake, fixed in https://github.com/netgen/site-bundle/pull/30/commits/742c29082ab9906a5ba11951f96aaab6c9e2661d